supervision/tracker/byte_tracker/core.py

```diff
@@ -281,6 +288,12 @@ def update_with_tensors(self, tensors: np.ndarray) -> List[STrack]:
         inds_second = np.logical_and(inds_low, inds_high)
         dets_second = bboxes[inds_second]
         dets = bboxes[remain_inds]
+        if masks is not None:
```

Comment:
Can we use `_keep`, and `_second` suffixes here? And make the new code consistent with the old one?